cob_sick_s300/ros/src/cob_sick_s300.cpp

```diff
@@ -291,7 +291,7 @@ class NodeClass
 			laserScan.header.frame_id = frame_id;
 			laserScan.angle_increment = vdAngRAD[start_scan + 1] - vdAngRAD[start_scan];
 			laserScan.range_min = 0.001;
-			laserScan.range_max = 30.0;
+			laserScan.range_max = 29.5; // though the specs state otherwise, the max range reported by the scanner is 29.96m
```

Comment:
To have a defined boundary. The max output of the scanner (I observed yesterday during some tests) was 2996mm, but due to the float conversion I figured it is better to have some clearance.

In the end, for localization/mapping tasks, we will probably limit the range anyways to something like 15m (at least indoors...)